content/docs/hello-world.md

```diff
@@ -10,14 +10,14 @@ El más pequeño de los ejemplos de React se ve así:
 
 ```js
 ReactDOM.render(
-  <h1>¡Hola mundo!</h1>,
+  <h1>Hello, world!</h1>,
```

Comment:
We missed this one!

Reply:
There is now also an edit for this in #174.